### ISSUE TITLE
Segmentation Fault in 'eco' (in a specific situation)

### ISSUE THREAD

**Issue by yashi21** [radare/radare2#4409]
In command prompt:

r2  [some_binary_file]
_CASE 1:_
->eco                                                                                                                                                   

> leading to segmentation fault, Ideally it should display available themes from _/usr/share/radare2/0.10.2-git/cons/_ 

_whereas
CASE 2:_
->eco basic                                                                                                                                         

> Changes to theme correctly

->eco     

> Correctly displays choices of themes available  

What's the issue?
Basically when I am using **eco** without specifying any theme it is leading to segmentation fault. This too happening only if eco  is used in such a way for the first time. If eco [theme] is run successfully once, then eco is running properly.

**Comment by radare** [comment]
Thanks! it's fixed now in master
